samples/blog/src/androidTest/kotlin/com/liferay/vulcan/blog/postings/activity/BlogPostingListTest.kt

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.vulcan.blog.postings.activity
+
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.assertion.ViewAssertions
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers
+import android.support.test.espresso.matcher.ViewMatchers.isDisplayed
+import android.support.test.espresso.matcher.ViewMatchers.withId
+import android.support.test.espresso.matcher.ViewMatchers.withText
+import android.support.test.filters.LargeTest
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+import android.view.WindowManager
+import com.github.kittinunf.result.Result
+import com.liferay.vulcan.blog.postings.R
+import com.liferay.vulcan.consumer.model.Thing
+import com.liferay.vulcan.consumer.requestParseWaitLoop
+import okhttp3.Credentials
+import okhttp3.HttpUrl
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class BlogPostingListTest {
+
+	@Rule
+	@JvmField
+	val activityRule = ActivityTestRule(MainActivity::class.java)
+
+	val credentials = Credentials.basic("vulcan@liferay.com", "vulcan")
+
+	@Before
+	fun unlockScreen() {
+		val activity = activityRule.getActivity()
+		val wakeUpDevice = Runnable {
+			activity.getWindow().addFlags(
+				WindowManager.LayoutParams.FLAG_TURN_SCREEN_ON or
+					WindowManager.LayoutParams.FLAG_SHOW_WHEN_LOCKED or
+					WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
+		}
+		activity.runOnUiThread(wakeUpDevice)
+	}
+
+	@Test
+	fun appRendersLayoutTest() {
+
+		val view = withId(R.id.thing_screenlet_activity)
+
+		onView(view).check(ViewAssertions.matches(ViewMatchers.isDisplayed()))
+	}
+
+	@Test
+	fun requestABlogFilteredByGroupId() {
+
+		val url = HttpUrl.parse("http://screens.liferay.org.es/o/api/p/blogs?id=57459&filterName=groupId")
+
+		url?.let {
+			val result: Result<Thing, Exception> = requestParseWaitLoop(url, mapOf(), listOf(), credentials)
+
+			Assert.assertNotNull(result.component1())
+
+			result.fold(success = {
+				Assert.assertEquals("http://screens.liferay.org.es/o/api/p/blogs", it.id)
+			}, failure = { Assert.fail() });
+		}
+	}
+
+	//	@Test
+	fun thingScreenletRenderingBlogsShowsResultsWithTextTest() {
```

Comment:
Why is this test commented out?

Reply:
I've updated the server to the latest vulcan dependency and the example.